packages/typegpu/src/data/attributes.ts

```diff
@@ -186,11 +210,41 @@ export function getCustomAlignment(
   data: AnyTgpuData | AnyTgpuLooseData,
 ): number | undefined {
   if (isDecorated(data) || isLooseDecorated(data)) {
-    return data.customAlignment;
+    return data.alignAttrib;
   }
   return undefined;
 }
 
+export function getCustomLocation(
+  data: AnyTgpuData | AnyTgpuLooseData,
+): number | undefined {
+  return data ? (data as unknown as BaseDecorated).locationAttrib : undefined;
```

Comment:
According to types the second branch of ternary will never run. Shouldn't we check for decorations to avoid casting?
```suggestion
  return isDecorated(data) || isLooseDecorated(data)
    ? data.locationAttrib
    : undefined;
```

Reply:
I'll remove the ternary, but the casting is mostly to reduce the runtime complexity, as accessing `locationAttrib` on an object that does not have it should result in `undefined` anyway, but is much less JS code than checking if the passed in value is a decorated object.

Reply:
Hmmm, okey. Then we should probably do the same for `getCustomAlignment` as it behaves nearly identically.